ItemService/ItemService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
```

Comment:
Can you please give more info for the purpose of this file? Thank you!

Reply:
removed